bin/configs/spring-cloud-3-with-optional.yaml

```diff
@@ -0,0 +1,14 @@
+generatorName: spring
+library: spring-cloud
+outputDir: samples/openapi3/client/petstore/spring-cloud-3-with-optional
```

Comment:
can you please update `.github/workflows/samples-spring.yaml` to include the new sample so that the CI will cover it moving forward?

Reply:
Done